index.html

```diff
@@ -123,11 +129,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
                 <div><img src="images/multi-post-stories2.png" alt="multi-post-stories2"></div>
                 <div class="tonic-heading">Multi-Post Stories</div>
```

Comment:
- [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`